Source/externs.m

```diff
@@ -267,22 +267,50 @@
 
 // Pasteboard Type Globals
 NSString *NSStringPboardType = @"NSStringPboardType";
+NSString *NSPasteboardTypeString = @"NSStringPboardType";
+
 NSString *NSColorPboardType = @"NSColorPboardType";
-NSString *NSFileContentsPboardType = @"NSFileContentsPboardType";
-NSString *NSFilenamesPboardType = @"NSFilenamesPboardType";
+NSString *NSPasteboardTypeColor = @"NSColorPboardType";
```

Comment:
Does something like `const NSString *` or `NSString const*` change anything? I'll merge as is, but consider looking at this in a future PR. Thanks :)